tox.ini

```diff
@@ -27,6 +27,8 @@ setenv =
     TF_GPU_ALLOCATOR=cuda_malloc_async
 commands =
     python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/core.git
+;    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git
+    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/dataloader.git@multiple_labels
```

Comment:
Testing with https://github.com/NVIDIA-Merlin/dataloader/pull/80. Should change to upstream before merging.